src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Main {
+    public static final int FIGURES_COUNT = 6;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        Figure[] figures = new Figure[FIGURES_COUNT];
+        for (int i = 0; i < figures.length / 2; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+
+        for (int i = figures.length / 2; i < figures.length; i++) {
+            figures[i] = figureSupplier.getDefaultFigure();
+        }
```

Comment:
Consider initializing your figures array with random figures in a single loop to make your code more concise and readable. You could use a ternary operator inside the loop to decide whether to use `getRandomFigure()` or `getDefaultFigure()` based on the current index.